tests/e2e/flowtest/flowtest_helpers.py

```diff
@@ -0,0 +1,223 @@
+        wait_for_storage_pods(timeout=300), 'Some pods were not in expected state'
+
+        logging.info("Node operation: Exit criteria verification: Success")
+
+    def add_capacity_entry_criteria(self):
+        """
+        Entry criteria verification function for add capacity operation
+        Returns:
+            tuple: containing the params used in add capacity operation
+
+        """
+        logger.info("Add capacity entry: Verifying cluster health")
+        self.sanity_helpers.health_check(cluster_check=False)
+        logger.info("Add capacity entry: Verifying StorageCluster pods are in running/completed state")
+        wait_for_storage_pods(timeout=300), 'Some pods were not in expected state'
+
+        logger.info("Add capacity entry: Getting restart count of pods before adding capacity")
+        restart_count_before = pod_helpers.get_pod_restarts_count(
+            defaults.ROOK_CLUSTER_NAMESPACE)
+
+        logger.info("Add capacity entry: Getting OSD pod count before adding capacity")
+        osd_pods_before = pod_helpers.get_osd_pods()
+
+        return osd_pods_before, restart_count_before
+
+    def add_capacity_exit_criteria(self, restart_count_before, osd_pods_before):
+        """
+        Exit criteria function for Add capacity operation
+        Args:
+            restart_count_before (dict): Restart counts of pods
+            osd_pods_before (list): List of OSD pods before
+
+        """
+        logger.info("Add capacity exit: Verifying cluster health")
+        self.sanity_helpers.health_check(cluster_check=False, tries=80)
+        logger.info("Add capacity exit: Verifying StorageCluster pods are in running/completed state")
+        wait_for_storage_pods(timeout=300), 'Some pods were not in expected state'
+
+        logger.info("Add capacity exit: Getting restart count of pods after adding capacity")
+        restart_count_after = pod_helpers.get_pod_restarts_count(
+            defaults.ROOK_CLUSTER_NAMESPACE)
+        logging.info(f"sum(restart_count_before.values()) = {sum(restart_count_before.values())}")
+        logging.info(f"sum(restart_count_after.values()) = {sum(restart_count_after.values())}")
+        assert sum(restart_count_before.values()) == sum(restart_count_after.values(
+        )), "Exit criteria verification FAILED: One or more pods got restarted"
+
+        osd_pods_after = pod_helpers.get_osd_pods()
+        number_of_osds_added = len(osd_pods_after) - len(osd_pods_before)
+        logger.info(f"Number of osds added = {number_of_osds_added}, "
+                    f"before = {len(osd_pods_before)}, after = {len(osd_pods_after)}")
+        assert number_of_osds_added == 3, "Exit criteria verification FAILED: osd count mismatch"
+
+        logging.info("Add capacity: Exit criteria verification: Success")
+
+
+@ignore_leftovers
+def create_pvc_delete(multi_pvc_factory, project=None):
```

Comment:
I think this function should be in sanity_helpers

Reply:
I couldn't find a proper one, there are functions similar to this but it also creates pods as well. My use case is just pvc create and delete. Also Pods will be interrupted during node failure operations.

Reply:
I meant that we should move this function to be a sanity_helpers method

Reply:
Thanks, moved